qcoverage/src/org/qcmg/coverage/Coverage.java

```diff
@@ -49,7 +47,7 @@ public Coverage(final Options options) throws Exception {
 	}
 
 	private void writePerFeatureTabDelimitedCoverageReport( final QCoverageStats stats) throws IOException {
-		final File file = new File(options.getOutputFileNames()[0]);
+		final File file = new File(options.getOutputFileNames()[0]+ ".txt");
```

Comment:
what if the output file name already has a ".txt" extension?

Reply:
so it will be output.txt.txt.  
The previous version coverage never checks the output file extension. 
I think *.txt.txt is OK for text output, *.txt.xml is better than *.txt for XML output, etc

Reply:
I think adding code to make this smarter is a good idea

Reply:
function added. However, our WDL pipeline has to be smart as well to predict the output file name.